gqueries/general/conversion/heat/network_gas_in_source_of_central_heat_production.gql

```diff
@@ -1,13 +1,16 @@
 # Direct conversion output of steam hot water from central heating through network gas heaters
 
 - query =
-    DIVIDE(
-      SUM(
-        V(
-          Q(central_heat_producing_converters_sankey),
-          "output_of_steam_hot_water * network_gas_input_conversion"
-        )
+    SUM(
+      DIVIDE(
+        SUM(
+          V(
+            Q(central_heat_producing_converters_sankey),
+            "output_of_steam_hot_water * network_gas_input_conversion"
+          )
+        ),
+        BILLIONS
       ),
-      BILLIONS
+      Q(gas_power_fuelmix_in_source_of_central_heat_conversion) * V(energy_mixer_for_gas_power_fuel, network_gas_input_conversion)
```

Comment:
If we're including the `network_gas` share in the `gas_power_fuelmix` here, shouldn't we similarly include the `crude_oil` share in the `oil_and_derivatives_in_source_of_central_heat_production` and the `bio_oil` share in `biomass_products_in_source_of_central_heat_production`?

Reply:
Good point, I've now added gas power fuelmix to the oil and biomass products queries.